Tools/wasm/emscripten/web_example/python.html

```diff
@@ -1,373 +1,433 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="author" content="Katie Bell">
-    <meta name="description" content="Simple REPL for Python WASM">
-    <title>wasm-python terminal</title>
-    <link rel="stylesheet" href="https://unpkg.com/xterm@4.18.0/css/xterm.css" crossorigin integrity="sha384-4eEEn/eZgVHkElpKAzzPx/Kow/dTSgFk1BNe+uHdjHa+NkZJDh5Vqkq31+y7Eycd"/>
-    <style>
-        body {
-            font-family: arial;
-            max-width: 800px;
-            margin: 0 auto
-        }
-        #code {
-            width: 100%;
-            height: 180px;
-        }
-        #info {
-            padding-top: 20px;
-        }
-        .button-container {
-            display: flex;
-            justify-content: end;
-            height: 50px;
-            align-items: center;
-            gap: 10px;
-        }
-        button {
-            padding: 6px 18px;
-        }
-    </style>
-    <script src="https://unpkg.com/xterm@4.18.0/lib/xterm.js" crossorigin integrity="sha384-yYdNmem1ioP5Onm7RpXutin5A8TimLheLNQ6tnMi01/ZpxXdAwIm2t4fJMx1Djs+"/></script>
-    <script type="module">
-class WorkerManager {
-    constructor(workerURL, standardIO, readyCallBack, finishedCallback) {
-        this.workerURL = workerURL
-        this.worker = null
-        this.standardIO = standardIO
-        this.readyCallBack = readyCallBack
-        this.finishedCallback = finishedCallback
-
-        this.initialiseWorker()
-    }
-
-    async initialiseWorker() {
-        if (!this.worker) {
-            this.worker = new Worker(this.workerURL, {type: "module"})
-            this.worker.addEventListener('message', this.handleMessageFromWorker)
-        }
-    }
-
-    async run(options) {
-        this.worker.postMessage({
-            type: 'run',
-            args: options.args || [],
-            files: options.files || {}
-        })
-    }
-
-    reset() {
-        if (this.worker) {
-            this.worker.terminate()
-            this.worker = null
-        }
-        this.standardIO.message('Worker process terminated.')
-        this.initialiseWorker()
-    }
-
-    handleStdinData(inputValue) {
-        if (this.stdinbuffer && this.stdinbufferInt) {
-            let startingIndex = 1
-            if (this.stdinbufferInt[0] > 0) {
-                startingIndex = this.stdinbufferInt[0]
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta name="author" content="Katie Bell" />
```

Comment:
I don't want to deny @katharosada credit here - but should this `author` be preserved? My concern is that given this is example code, the likelihood of the `<head>` block being copy-pastad into other projects seems high. Looking at other code in the Python codebase, the only places a specific name is called out is in a copyright assertion; and there, it's always in a comment, rather than in something that is "externally visible" as it would be here. 

Unless there's a credit or copyright assertion reason for preserving this, I think we'd be better served removing the author meta tag - but I don't want to do that without @katharosada's informed consent.

Reply:
I agree it might make sense to remove this author tag. But shouldn't this be done in a separate PR? I'd like to keep all the automatic noisy formatting changes by themselves?

Reply:
Fair enough - I can live with that.

Reply:
Pretty sure that author tag was added automatically by my IDE or some plugin when I created the HTML file, I didn't even notice it.
Feel free to remove, no argument here!